docker-hub/image-access-management.md

```diff
@@ -39,19 +39,50 @@ To ensure that each org member uses images in a safe and secure environment, you
     > **Note**
     >
     > There currently is no Docker Desktop for Linux. Linux users will have the same restrictions as Mac and Windows users while logged in, but there is currently no way to require them to log in.
-  
-2. On Windows, create a file `C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (i.e., only the system administrator can write to the file). The file must be JSON and contain one or more organization names in the `allowedOrgs` key. The user must sign in and be a member of at least one of the organizations before using Docker Desktop. For example:
 
-    ```json
+2. Create a registry.json file on Windows or Mac.
+
+For **Windows**:
+
+Create a file `C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (i.e., only the system administrator can write to the file). The file must be JSON and contain one or more organization names in the `allowedOrgs` key.
+
+To create your registry.json file:
+
+- Open Windows Powershell and select Run as Administrator.
+- Type the following command: `cd /ProgramData/DockerDesktop/`
+- In notepad, type `registry.json` and enter one or more organization names in the `allowedOrgs` key and click Save.
+
+Example:
+
+        ```json
```

Comment:
Needs formatting. Could you try removing the indent?

![image](https://user-images.githubusercontent.com/47779042/135661226-ce01a24f-02e9-4493-993e-6c17073f8874.png)